build-ios.bash

```diff
@@ -7,7 +7,7 @@ go get -u golang.org/x/mobile/cmd/gomobile
 gomobile init
 output=MOBILE/ios/oonimkall.framework
 gomobile bind -target=ios -o $output -ldflags="-s -w" ./pkg/oonimkall
-release=$(git describe --tags)
+release=$(git describe --tags || echo $GITHUB_SHA)
```

Comment:
We don't have `git` inside GitHub actions. We are currently not _publishing_ packages, so it's fine for us to use the GITHUB_SHA environment variable for now.